lascli/actions/workflows.py

```diff
@@ -0,0 +1,262 @@
+import requests
+import traceback
+import uuid
+
+from datetime import datetime
+from las import Client
+from typing import Optional
+
+from ..util import wrap_output, capture_return
+
+
+# FIXME: Get this from somewhere else.
+COMPONENT_ASSEST_URI = 'https://raw.githubusercontent.com/LucidtechAI/las-docs/master/tutorials/simple-demo/backend/src/Invoice/assets/jsRemoteComponent.js'  
+
+created_ids = collections.defaultdict(list)
+ 
+
+def create_workflow_spec(
+    model_id: str, 
+    preprocess_transition_id: str, 
+    manual_transition_id: str,
+    postprocess_transition_id: str, 
+    tag: str,
+):
+    return {
+        'definition': {
+            'Comment': f'Auto-generated workflow for model {model_id}. {tag}.',
+            'StartAt': 'Preprocess',
+            'States': {
+                'Preprocess': {
+                    'Type': 'Task',
+                    'Resource': preprocess_transition_id,
+                    'Next': 'CheckValidation'
+                },
+                'CheckValidation': {
+                    'Type': 'Choice',
+                    'Choices': [{
+                        'Variable': '$.needsValidation',
+                        'BooleanEquals': False,
+                        'Next': 'Postprocess'
+                    }, {
+                        'Variable': '$.needsValidation',
+                        'BooleanEquals': True,
+                        'Next': 'Validate'
+                    }]
+                },
+                'Validate': {
+                    'Type': 'Task',
+                    'Resource': manual_transition_id,
+                    'Next': 'Postprocess'
+                },
+                'Postprocess': {
+                    'Type': 'Task',
+                    'Resource': postprocess_transition_id,
+                    'End': True
+                }
+            }
+        }
+    }
+    
+
+def create_field_config(las_client: Client, model_id: str):
+    model = las_client.get_model(model_id)
+    field_config = model['fieldConfig']
+
+    def transform(field):
+        return {
+            'type': field_config[field]['type'],
+            'display': field_config[field].get('description') or field,
+            'confidenceLevels': {
+                'automated': 0.95,
+                'high': 0.80,
+                'medium': 0.5,
+                'low': 0.3,
+            }
+        }
+
+    return {field: transform(field) for field in field_config}
+    
+
+@wrap_output(start_msg='Creating secrets ...', end_msg='Done')
+@capture_return(dest=created_ids['secrets'])
+def create_secrets(las_client: Client, create_tag: str, username: str = None, password: str = None):
+    docker_secret = {}
+    if username and password:
+        docker_secret = las_client.create_secret(
+            data={
+                'username': username,
+                'password': password 
+            },
+            name='Docker credentials',
+            description=create_tag,
+        )['secretId']
```

Comment:
Looks like you are rewriting a dict to a string, is this intentional? This will probably fail in the return statement when trying to `.get('secretId')` on a string

```suggestion
    docker_secret = {}
    if username and password:
        docker_secret = las_client.create_secret(
            data={
                'username': username,
                'password': password 
            },
            name='Docker credentials',
            description=create_tag,
        )['secretId']
```